plugins/notifications-backend/README.md

```diff
@@ -44,7 +52,15 @@ In the psql prompt use these commands for creating the table:
 
 ```sql
 \c backstage_plugin_notifications
-create table messages (id SERIAL, subject VARCHAR(255), body TEXT);
+create extension if not exists "uuid-ossp";
+drop table if exists messages, links, labels;
+
+create table messages (id uuid DEFAULT uuid_generate_v4(), kind VARCHAR(255) DEFAULT 'Notification', created TIMESTAMP DEFAULT CURRENT_TIMESTAMP, origin VARCHAR(255), subject VARCHAR(255) NOT NULL, body TEXT, PRIMARY KEY (id));
```

Comment:
We will probably come back to a simple int sequence instead of uuids. For simplicity

Reply:
do you think that an int will be acceptable? cause if it is then I am in favor of going back to int.

Reply:
I believe so. The UUIDs would be a nice addition but nothing requires it, AFAIK.